src/lib/create-app.ts

```
@@ -22,7 +24,14 @@ export default function createApp() {
       credentials: true,
     })
   );
-
+  app.use("*", async (c, next) => {
+    cloudflareRateLimiter<AppBindings>({
+      rateLimitBinding: (c) => c.env.MY_RATE_LIMITER,
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)